h2o-algos/src/main/java/hex/gam/MatrixFrameUtils/GamUtils.java

```diff
@@ -143,7 +147,27 @@ public static GLMParameters copyGAMParams2GLMParams(GAMParameters parms, Frame t
     return glmParam;
   }
 
-  public static void setParamField(GAMParameters parms, GLMParameters glmParam, boolean superClassParams, Field[] gamFields) {
+  public static void setParamField(Model.Parameters parms, GLMParameters glmParam, boolean superClassParams,
+                                   Field[] gamFields, List<String> excludeList) {
+    // assign relevant GAMParameter fields to GLMParameter fields
+    Field glmField;
+    boolean emptyExcludeList = excludeList.size() == 0;
+    for (Field oneField : gamFields) {
+      try {
+        if (emptyExcludeList || !excludeList.contains(oneField.getName())) {
+          if (superClassParams)
+            glmField = glmParam.getClass().getSuperclass().getDeclaredField(oneField.getName());
+          else
+            glmField = glmParam.getClass().getDeclaredField(oneField.getName());
+          glmField.set(glmParam, oneField.get(parms));
+        }
+      } catch (IllegalAccessException|NoSuchFieldException e) { // suppress error printing, only cares about fields that are accessible
+        ;
+      }
+    }
+  }
+  
+/*  public static void setParamField(GAMParameters parms, GLMParameters glmParam, boolean superClassParams, Field[] gamFields) {
```

Comment:
possibly could be deleted since it is commented out

Reply:
Yes, I deleted it.  Thanks.